1.4.0 release

KingLemming opened the pull request.

covers1624 commented:
Sorry, we are done with 1.7.

vinyldarkscratch commented:
Oh whoops, didn't mean to commit this particular change, wrong mod!  More of that whole mod updating thing I was telling you about.  :P

KingLemming committed:
Add Gourmand and Disenchantment dynamos.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Dynamo energy configuration.

KingLemming committed:
Machine power configs.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Block and Item asset restructuring.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Container refactorizin'

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Restructure lang files.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Texture reorganization.
Fix 2 bugged Smelter recipes.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Update accesstransformer.cfg
Adjust sap recipe. Add bottler recipe for syrup.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Refactor Core module workspace.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add Slot Seal item and support.

Signed-off-by: King Lemming <kinglemming@gmail.com>

covers1624 committed:
Mavenize!

- Move to official mappings.
- Prepare for Maven.
- Refactor generated resources.

KingLemming committed:
Mod tagging. :)

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
AccessTransformer update.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Update README
Add corrected loot tables.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fluid ingredient work. -_-

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
New centrifuge recipes.

Signed-off-by: King Lemming <kinglemming@gmail.com>

jaredlll08 committed:
Add Disenchantment JEI category

KingLemming committed:
Merge pull request #1 from jaredlll08/dev-fluid-ingredient

Add Disenchantment JEI category

KingLemming committed:
Refactors.

Signed-off-by: King Lemming <kinglemming@gmail.com>

karakufire committed:
create ja_jp.json

KingLemming committed:
Fix mcmeta file.
Add Disenchantment and Gourmand dynamo recipes and JEI handlers.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Adjustments for new recipe tooltips.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Add Netherite recipes. Remove item cell.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Fix Pyrolyzer model not being generated

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Beta release staging.

Signed-off-by: King Lemming <kinglemming@gmail.com>

karakufire committed:
update ja_jp.json

KingLemming committed:
Move Blocks to proper Creative Tabs.

Signed-off-by: King Lemming <kinglemming@gmail.com>

KingLemming committed:
Merge pull request #2 from karakufire/ja_jp

add ja_jp.json
